hw-model/src/output.rs

```diff
@@ -79,6 +79,9 @@ impl OutputSink {
     pub fn set_now(&self, now: u64) {
         self.0.now.set(now);
     }
+    pub fn get_now(&self) -> u64 {
```

Comment:
nitpick: In idiomatic Rust, getters don't prefix get_:

https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter

Reply:
Removed